packages/dumb/src/models/stats.model.ts

```diff
@@ -57,3 +58,11 @@ export interface BgClasses {
   target: number // Target threshold represents all the values between low and target
   veryLow: number // Very low threshold represents all the values between 0 and veryLow
 }
+
+export enum StatFormats {
```

Comment:
To be consistent with (a lot of) other enums, the enum name should be singular (`StatFormat`) and the values should be PascalCase (`Cv`, `Gmi`, etc.)